rs/registry/canister/src/mutations/node_management/do_update_node_ipv4_config_directly.rs

```diff
@@ -221,8 +181,8 @@ mod tests {
                 .expect("failed to get the node operator id");
 
         // create IPv4 config with invalid IP address
-        let mut ipv4_config = init_ipv4_config();
-        ipv4_config.ip_addr = "193.118.256.140".into();
+        let ipv4_config = init_ipv4_config();
+        IPv4Config::new("193.118.256.140".into(), "193.118.59.137".into(), 29).unwrap();
```

Comment:
This doesn't properly test it: the test succeeds because of the second line (185) that throws the error, but actually the error should come from line 192.
```suggestion
        let ipv4_config = IPv4Config::new("193.118.256.140".into(), "193.118.59.137".into(), 29).unwrap();
```

Reply:
sorry my mistake